### PR TITLE
Fix: ESLint on ./docs

### LINKED ISSUE #664: Fix ESLint on Docs
ESLint is not running on the docs. Whilst we don't care about coverage on these files it'd be nice to have syntax consistency until we can get prettier in.

- [x] Exclude examples because of formatting issues.

### PR DESCRIPTION
**Pull request by omgaz**
ESLint was not running on docs.
ESLint should run on docs.
ESLint should still ignore examples (because of formatting requirements).

Resolves #664

### CODE REVIEW

File: docs/components/Contributors/index.jsx [hunk 1]
```diff
@@ -8,25 +8,23 @@ import {
 
 import './styles.scss';
 
+function getContributors() {
```
Comment (omgaz) [inline docs/components/Contributors/index.jsx]: getContributors should be a static method.

**Comment by ChaoLiangSuper** [comment]
Maybe also need to apply scss lint to doc folder as well?
Current `"sass-lint": "sass-lint './src/**/*.scss' -v",`

File: docs/components/Contributors/index.jsx (hunk 1 above)
Comment (lightbringer1991) [inline docs/components/Contributors/index.jsx]: is it because eslint complains that this function can be static?

**Comment by omgaz** [comment]
@ChaoLiangSuper https://github.com/Adslot/adslot-ui/issues/666

**Pull request merged**